pdal/SpatialReference.cpp

```diff
@@ -196,7 +196,7 @@ std::string SpatialReference::getVerticalUnits() const
 
     std::string wkt = getVertical();
     const char* poWKT = wkt.c_str();
-    OGRScopedSpatialReference poSRS = ogrCreateSrs(m_wkt);
+    OGRScopedSpatialReference poSRS = ogrCreateSrs(poWKT);
```

Comment:
This would likely be an intentional or unintentional behavior change.

Reply:
wkt / poWKT was unused before. It makes more sense for a getVerticalUnits() method to use the wkt of getVertical() right ?